### ISSUE TITLE
Is it possible to make the calendar updates in real time?

### ISSUE THREAD

**Issue by voidplayer** [etesync/etesync-dav#179]
Is there a way for the server to push updates?

Right now, when people collaborate in a calendar dates could overlap due to the fact that they updates are not instantenous

**Comment by tasn** [comment]
You probably just need to configure your client to fetch more often. EteSync-DAV has a little throttling enabled (2 minutes), though it's more likely that your calendar client is not syncing as often.

**Comment by voidplayer** [comment]
Yeah, i know i can probably down the fetching time. But i cant help but feeling that maybe theres a 3-4h gap where everybody is going to be using it and i dont need to put that stress on the server by polling it every 30 secs 8h during every night

I was thinking of using a web version or something where everybody can look at an updated version, or maybe have some push updates

Maybe i worry too much and its not that much of a stress having all the clients polling that often

**Comment by tasn** [comment]
You can also just fetch manually when working on it with others in a real-time crunch. Though yeah, there's also the web client.

It's less than optimal having everyone polling the server every 30 seconds (which is why we throttle the DAV client to a poll every 2 minutes), but it's not the end of the world if you really need it.
